src/platform/common/application/applicationEnvironment.node.ts

```diff
@@ -5,19 +5,20 @@
 
 import { inject, injectable } from 'inversify';
 import * as path from 'path';
-import * as vscode from 'vscode';
 import { IPlatformService } from '../platform/types';
 import { IExtensionContext, IPathUtils } from '../types';
 import { OSType } from '../utils/platform';
-import { Channel, IApplicationEnvironment } from './types';
+import { BaseApplicationEnvironment } from './applicationEnvironment.base';
 
 @injectable()
-export class ApplicationEnvironment implements IApplicationEnvironment {
+export class ApplicationEnvironment extends BaseApplicationEnvironment {
```

Comment:
THis was refactored into two classes.
One web, and one node.

The node one supports returning a userSettingsFile and a userCustomKeybindingsFile. The web class will return undefined for these two items (for now)